### PR TITLE
Add support for selected MusicBrainz identifiers

### PR DESCRIPTION
**Pull request by uklotzde**
- Only the single-valued recording/track/album/release identifiers, no artist identifiers.
- No mapping of `MusicBrainzRecordingId` for ID3v2 yet, see #160.

### CODE REVIEW

File: src/tag/item.rs [hunk 1]
```diff
@@ -208,7 +212,10 @@ gen_map!(
 	"TKWD"                  => PodcastKeywords,
 	"COMM"                  => Comment,
 	"TLAN"                  => Language,
-	"USLT"                  => Lyrics
+	"USLT"                  => Lyrics,
+	"MusicBrainz Release Track Id" => MusicBrainzTrackId,
+	"MusicBrainz Album Id"  => MusicBrainzReleaseId,
+	"MusicBrainz Release Group Id" => MusicBrainzReleaseGroupId
```
Comment (Serial-ATA) [inline src/tag/item.rs:218]: Could you fixup the alignment of the arrows here?

File: src/tag/item.rs [hunk 2]
```diff
@@ -278,7 +285,11 @@ gen_map!(
 	"----:com.apple.iTunes:LANGUAGE"              => Language,
 	"----:com.apple.iTunes:SCRIPT"                => Script,
 	"\u{a9}lyr"                                   => Lyrics,
-	"xid "                                        => AppleXid
+	"xid "                                        => AppleXid,
+	"----:com.apple.iTunes:MusicBrainz Track Id"  => MusicBrainzRecordingId,
+	"----:com.apple.iTunes:MusicBrainz Release Track Id" => MusicBrainzTrackId,
+	"----:com.apple.iTunes:MusicBrainz Album Id"  => MusicBrainzReleaseId,
+	"----:com.apple.iTunes:MusicBrainz Release Group Id" => MusicBrainzReleaseGroupId
```
Comment (Serial-ATA) [inline src/tag/item.rs:292]: Same as above

**Review by Serial-ATA — CHANGES_REQUESTED** [review]

File: src/tag/item.rs (hunk 1 above)
Comment (uklotzde) [reply src/tag/item.rs:218]: That's why I prefer **not** to use any manual alignment. It causes changes in totally unrelated and unchanged lines.

**Comment by Serial-ATA** [comment]
Thanks!

**Pull request merged**